ios-cr-task/cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
```

Comment:
This comment should be deleted. It doesnt have any relation with this file. Instead of that, you can have doc comment which will describe the usage of this file.

Reply:
Also, it would be better, to have the file name, at the top of the file, author name, creation date, and the file owner information. like in suggestion
```suggestion
//
//  PaymentViewController.swift
//  Payment Modul
//
//  Created by Tigran Simonyan on 05.01.22.
//  Copyright © 2022 Scriptomania. All rights reserved.
//
```